docs_src/microservices/application/services/AppServiceConfigurable/ApiReference.md

```diff
@@ -2,4 +2,4 @@
 
```

Comment:
should these headers be combined? also does this page need a 
```
---
title: App Service Configurable - APIs
---
```

Reply:
Out of scope for this PR. Will address these titles in next PR to refactor all the the existing refactored services.